omdata/orca/recipes.py

```diff
@@ -112,22 +123,34 @@ def ase_relaxation(
         Dictionary of optimizer parameters
     outputdir: str
         Directory to move results to upon completion
+    calc kwargs: dict
```

Comment:
Technically, this is not a `dict`. The user can pass in any keyword arguments they want but not as dict format.